lib/amorail/property.rb

```diff
@@ -4,15 +4,19 @@ module Amorail
   # Return hash key as method call
   module MethodMissing
     def method_missing(method_sym, *arguments, &block)
-      if data.key?(method_sym.to_s)
-        data.fetch(method_sym.to_s)
+      if data.key?(prep_method_name(method_sym))
+        data.fetch(prep_method_name(method_sym))
       else
         super
       end
     end
 
     def respond_to_missing?(method_sym, *args)
-      args.size.zero? && data.key?(method_sym.to_s)
+      args.size.zero? && data.key?(prep_method_name(method_sym))
```

Comment:
@palkan Please take a look at this. I understand that is standard case usind **respond_to missing?**, but I could make mistake because I have little experience in ruby/ror.

Reply:
Yeah, that looks good.
Only one suggestion: let's not extract `to_s.downcase` to a `prep_method_name` to avoid adding one more method to the module. It's OK to duplicate here.

Reply:
Ok, done.